### DISCUSSION TITLE
the results discussion

### DISCUSSION THREAD

**Discussion by zc994646659** [mailhexu/TB2J#10] [Q&A]
I used VASP + wannier90 to get a set of data. Should I consider the problem of projection? The magnetic moment calculated by VASP is 4ub, while the Wannier projection is only 2.5. Another problem is that there are two interactions with almost no difference in distance (difference 0.001 Å), and the magnitude of the interaction is 4 times different. Do you think it's strange? I am not very proficient in the use of Wannier 90. Thank you for giving me your suggestions on the above problems.

**Comment by zc994646659** [comment]
[exchange.txt](https://github.com/mailhexu/TB2J/files/6936668/exchange.txt)
this is  the result

**Comment by mailhexu** [comment]
Hi, 
Often, this means the Wannier functions are not well constructed or the fermi energy is not correctly specified in the wann2J.py command. I guess it's the Fermi energy issue, as from the exchange.txt the charges are too small. Could you check that?
Best regards,
HeXu

**Comment by zc994646659** [comment]
Yes , the charge projected is too small,the results calculated by vasp is nearly 5,so how to deal with it, what you mean is to change the wann2j.py to correct the Fermi energy?------------------&nbsp;原始邮件&nbsp;------------------
***@***.***&gt;
发送时间:&nbsp;2021年8月5日(星期四) 下午4:01
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [mailhexu/TB2J] the results discussion (#10)

**Comment by mailhexu** [comment]
Find the Fermi energy from the VASP output, and in the wann2J.py command, use the --efermi option to specify the fermi energy. Note that the charge in VASP include all the s, p, d... orbitals, while not all of them are included in the Wannier function so the charge could be quite different. The one you should be looking for is the d orbital occupation for Mn. I guess that is around 4 or 5.
